adguard/DOCS.md

```diff
@@ -30,6 +30,7 @@ comparison to installing any other Home Assistant add-on.
 1. Start the "AdGuard Home" add-on.
 1. Check the logs of the "AdGuard Home" to see if everything went well.
 1. Click the "OPEN WEB UI" button and log in with your Home Assistant account.
+1. If you have set your router to use the IP address of your Home Assistant as its DNS, you need to add the router's IP address as a client in AdGuard's settings under _Settings → Client settings_.
```

Comment:
Oh hi there @tomayac 👋 

That is not really needed and depends on your configuration and network setup. It is definitely not needed to get the add-on installed and running, which is what these instructions are about.

For that reason, I'm going to friendly decline this addition. Nevertheless, thanks for being willing to contribute 👍 

../Frenck

Reply:
Understand, and fair enough. It might still make sense to link users out to some next steps guide then. The linked forum post was the first result on Google for “adguard home assistant [autocomplete: not working]”, so I would argue some guidance is still needed.

Reply:
That guidance is The AdGuard Home documentation. Which is also linked in the application itself. As a matter of fact, they have a dedicated set up tab in their application for that.